### PR TITLE
Add justchunks.net

### PR DESCRIPTION
**Pull request by scrayos**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/justchunks/manifest.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+  "supported_languages": [
+    "de",
+    "en",
+    "es"
+  ],
+  "social": {
+    "web": "https://justchunks.net",
+    "web_shop": "https://shop.justchunks.net",
+    "web_support": "https://forum.timolia.de",
```
Comment (JakobBruening) [inline minecraft_servers/justchunks/manifest.json:16]: This is not correct I guess? 🤔

File: minecraft_servers/justchunks/manifest.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "location": {
+    "city": "Falkenstein",
+    "country": "Germany",
+    "country_code": "DE"
+  },
+  "yt_trailer": "Ngi6n0XI2dM",
+  "user_stats": "https://justchunks.net/user/{userName}",
+  "discord": {
+    "server_id": 140450742329671680,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/justchunks/manifest.json:54]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/justchunks/manifest.json (hunk 1 above)
Comment (scrayos) [reply minecraft_servers/justchunks/manifest.json]: Thanks, this one slipped through :sweat_smile:

File: minecraft_servers/justchunks/manifest.json (hunk 2 above)
Comment (scrayos) [reply minecraft_servers/justchunks/manifest.json]: Ah, thanks for the info. I've changed it to `false` :+1:

**Comment by scrayos** [comment]
Okay, I've implemented all the feedback and have rebased the branch on master.

**Comment by scrayos** [comment]
@JakobBruening Okay, I've implemented all the feedback and have rebased the branch on master.

**Review by JakobBruening — APPROVED** [review]

**Pull request merged**